### PR TITLE
goodbye azure

### PR DESCRIPTION
**Pull request by reaperhulk**

### CODE REVIEW

File: MANIFEST.in [hunk 1]
```diff
@@ -16,8 +16,7 @@ recursive-include tests *.py
 exclude vectors
 recursive-exclude vectors *
 
-exclude azure-pipelines.yml .azure-pipelines .travis.yml .travis
-recursive-exclude .azure-pipelines *
+exclude .travis.yml .travis
```
Comment (alex) [inline MANIFEST.in:19]: Probably need to add `.github` here?

File: release.py [hunk 2]
```diff
@@ -14,9 +14,6 @@
 import time
 import zipfile
 
-from azure.devops.connection import Connection
-from azure.devops.v5_1.build.models import Build
-
 import click
 
 from msrest.authentication import BasicAuthentication
```
Comment (alex) [inline release.py:19]: This can go as well.

File: tox.ini [hunk 3]
```diff
@@ -8,7 +8,7 @@ extras =
     test
     idna: idna
 deps =
-    # This must be kept in sync with .travis/install.sh and azure-pipelines.yml
+    # This must be kept in sync with .travis/install.sh and GitHub Actions ci.yml
```
Comment (alex) [inline tox.ini:11]:
```suggestion
    # This must be kept in sync with .travis/install.sh and .github/workflows/ci.yml
```

File: MANIFEST.in (hunk 1 above)
Comment (reaperhulk) [reply MANIFEST.in:19]: Done, although this probably means we've been packaging .github for a few releases 😆

**Comment by alex** [comment]
> Can you remove the azure reference in `.travis/install.sh`

still outstanding

**Comment by alex** [comment]
flake8 error

**Pull request merged**